frontend/src/pages/Partners/Partners.tsx

```diff
@@ -0,0 +1,225 @@
+import euLogo from './european-social-fund.png';
+import Meta from '@components/Meta/Meta';
+
+const links = [
+  { cd: { logo: ceskoDigitalLogo, href: 'https://cesko.digital/' } },
+  { nadacecs: { logo: nadaceCsLogo, href: 'https://www.nadacecs.cz/' } },
+  { blizksobe: { logo: blizKSobeLogo, href: 'https://www.blizksobe.cz/' } },
+  { zemekvet: { logo: zemeKvetLogo, href: 'http://zemekvet.cz/' } },
+  { mhmp: { logo: mhmpLogo, href: 'https://granty.praha.eu/GrantyPortal/default' } },
+  { otevreno: { logo: otevrenoLogo, href: 'https://otevreno.org/' } },
+  { ucitelnazivo: { logo: ucitelNazivoLogo, href: 'https://www.ucitelnazivo.cz/' } },
+  { up: { logo: ucitelskaPlatformaLogo, href: 'https://www.ucitelskaplatforma.cz/' } },
+  { skav: { logo: skavLogo, href: 'https://skav.cz/' } },
+  { edu: { logo: eduLogo, href: 'https://edu.cz/' } },
+  { nadacekj: { logo: nadaceKarlaJaneckaLogo, href: 'https://www.nadacekj.cz/' } },
+  { vodafone: { logo: laboratorVodafoneLogo, href: 'https://www.nadacevodafone.cz/' } },
+  { eu: { logo: euLogo } },
+];
+const Partners: React.FC = () => (
+  <Container>
+    <Meta
+      title="Přispějte na projekt Začni učit a zlepšete české školství"
+      description="Chcete nám s naší prací pomoci nebo nás finančně podpořit? Ozvěte se na info@vyluka.org a společně to vymyslíme!"
+    />
+    <BackButton href={routes.homepage} icon="home">
+      Zpátky domů
+    </BackButton>
+
+    <Section>
+      <MainTitle>Partneři</MainTitle>
+
+      <Paragraph>
+        Za webovou platformou Začni učit! stojí{' '}
+        <MainLink className="underline" href={routes.vyluka}>
+          {' '}
+          organizace Výluka, z.s.
+        </MainLink>
+      </Paragraph>
+
+      <Paragraph>
+        <strong>Chcete s námi spolupracovat či přispět k naší práci?</strong>
+      </Paragraph>
+
+      <Paragraph>Chcete nám pomoct finančně nebo s naší prací?</Paragraph>
+
+      <Paragraph>
+        Ozvěte se na{' '}
+        <MainLink className="underline" href="mailto:info@vyluka.org">
+          info@vyluka.org
+        </MainLink>{' '}
+        a společně to vymyslíme!
+      </Paragraph>
+      <H2>S kým spolupracujeme?</H2>
+    </Section>
+    <Section className="bgr">
+      <Link href={links[0].cd.href} target="_blank">
+        <Logo src={links[0].cd.logo} alt="" />
+      </Link>
+      <Paragraph>
+        Dobrovolníci sdružení na platformě{' '}
+        <MainLink href={links[0].cd.href} target="_blank">
+          Česko.Digital
+        </MainLink>{' '}
+        pomohli proměnit v realitu naši představu o webu, který provádí zájemce o učitelskou kariéru
+        na celé jejich cestě za katedru.{' '}
+      </Paragraph>
+    </Section>
+    <Section>
+      <Link href={links[1].nadacecs.href} target="_blank">
+        <Logo src={links[1].nadacecs.logo} alt="" />
+      </Link>
+      <Paragraph>
+        <MainLink href={links[1].nadacecs.href} target="_blank">
```

Comment:
ten obsah paragraphu si přidej do toho links pole (možná to pak přejmenuj na data?) co máš nahoře. A tady použij cyklus. Ať nemusíš duplikovat tu html strukturu

Reply:
Opraveno :)